paddle/fluid/operators/math/sequence_scale.h

```diff
@@ -46,7 +46,7 @@ namespace math {
 template <typename DeviceContext, typename T>
 class ScaleLoDTensorFunctor {
  public:
-  void operator()(const DeviceContext& context, framework::LoDTensor& seq,
+  void operator()(const DeviceContext& context, framework::LoDTensor* seq,
                   const T* scales);
```

Comment:
Please pay attention to the parameter order.
> When ordering function parameters, put all input-only parameters before any output parameters. I

https://google.github.io/styleguide/cppguide.html#Output_Parameters

Reply:
Done. Thank you for the feedback.